### PR TITLE
Fix for Issue-07

### PR DESCRIPTION
**Pull request by shivamprakash**
Check for ._ in the filename

### CODE REVIEW

**Comment by erickpeirson** [comment]
@shivamprakash, this looks like a reasonable fix. But where are the tests? You should add some tests to show that your fix solves the problem. So in this case, you might have a test dataset polluted by those pesky ``._`` files, and your test might be to ingest that dataset and assert that only the desired files have been added.

Make sense?

**Comment by erickpeirson** [comment]
Oh, and be sure to reference the issue in your commit messages and pull request messages, like this: issue #7

**Comment by erickpeirson** [comment]
@shivamprakash Let me know when you'd like for me to look at this. Looks like you're still working on it?

**Comment by shivamprakash** [comment]
Yeah sure.  I am writing test cases for bulk upload. Once it is done, I will push them.

**Comment by erickpeirson** [comment]
@shivamprakash Great! Comment here when your tests are passing and you're ready to merge.

File: jars/cookies/content.py [hunk 1]
```diff
@@ -68,10 +72,8 @@ def handle_bulk(file, form):
 
     # Each file will result in a new LocalResource.
     for name in z.namelist():
-    
         # Some archives have odd extra files, so we'll skip those.
-        if not name.startswith('._'):
-        
+        if not basename(name).startswith('._'):
```
Comment (shivamprakash) [inline jars/cookies/content.py]: This is the fix. FIxing it by checking the basename of the file has ._ .
If not then only consider those files.

**Pull request closed without merging**